### PR TITLE
Skew Normal Truncation Term Bugfix

### PR DESCRIPTION
**Pull request by Comack**
Conform to requirement that inputs passed into PyTorch's log() function must be tensors.

### CODE REVIEW

File: src/stream_ml/pytorch/builtin/_skewnormal.py [hunk 1]
```diff
@@ -124,7 +126,7 @@ def log_truncation_term(
     """Log of integral from a to b of skew-normal."""
     erfa = xp.erf(skew * (ab[0] - loc) / sigma / _sqrt2)  # type: ignore[attr-defined]
     erfb = xp.erf(skew * (ab[1] - loc) / sigma / _sqrt2)  # type: ignore[attr-defined]
-    return xp.log(erfb - erfa) + xp.log(erfb + erfa + 2) - xp.log(4)
+    return xp.log(erfb - erfa) + xp.log(erfb + erfa + 2) - xp.log(Tensor(4.0))
```
Comment (nstarman) [inline src/stream_ml/pytorch/builtin/_skewnormal.py:129]: It's probably easier to do

```suggestion
    return xp.log(erfb - erfa) + xp.log(erfb + erfa + 2) - math.log(4.0)
```

Comment (Comack) [reply src/stream_ml/pytorch/builtin/_skewnormal.py]: Agreed.

**Comment by nstarman** [comment]
There's a git conflict in this PR.
Also, I believe you did this PR from your main, which means I can't edit the PR.
PRs should originate from feature branches so they don't mess up your main branch. 
After this is merged you probably want to rewrite your git history so that your main branch matches the main branch of this repo.

**Comment by Comack** [comment]
Thanks for letting me know, I think I might've set the remote incorrectly.

**Comment by nstarman** [comment]
Great! Merging.

**Pull request merged**